modules/apps/dynamic-data-mapping/dynamic-data-mapping-taglib/src/main/resources/META-INF/resources/template_selector/page.jsp

```diff
@@ -26,14 +26,15 @@ Group ddmTemplateGroup = GroupLocalServiceUtil.getGroup(ddmTemplateGroupId);
 %>
 
 <clay:content-row
+	floatElements=""
 	verticalAlign="center"
 >
 	<clay:content-col
 		cssClass="inline-item-before"
 	>
 		<aui:input id="displayStyleGroupId" name="preferences--displayStyleGroupId--" type="hidden" value="<%= String.valueOf(displayStyleGroupId) %>" />
 
-		<aui:select id="displayStyle" inlineField="<%= true %>" label="<%= HtmlUtil.escape(label) %>" name="preferences--displayStyle--">
+		<aui:select id="displayStyle" label="<%= HtmlUtil.escape(label) %>" name="preferences--displayStyle--" wrapperCssClass="c-mb-4">
```

Comment:
I tried locally with the navigation-menu widget, but since the input field has already a margin the new one seems too much

![image](https://user-images.githubusercontent.com/46778114/89921392-edbcbe80-dbfd-11ea-8c01-0decee6f45f2.png)

Reply:
the new one has 4px more on mobile, just not to misalign

Reply:
I don't know what you mean but I guess the navigation has changed and I'm not seeing it 🤷‍♂️